game_server/src/webhogg_game.rs

```diff
@@ -0,0 +1,13 @@
+use crate::maths::Vec2;
+
+pub struct WebhoggPlayer {
+    pos: Vec2,
+}
+
+pub struct WebhoggGame {
```

Comment:
Does a WebhoggGame really always have exactly two WebhoggPlayers (also at startup)? Don't forget, Rust does not have null-types. In that case, Options would be the soulution. But of course I'm not aware of your plans for this structure.
But in general: you want the WebhoggGame and the WebhoggPlayer having a constructor function.